ScoutSuite/core/server.py

```diff
@@ -0,0 +1,107 @@
+from sqlitedict import SqliteDict
```

Comment:
Copied from #260 , @Aboisier :
> This class lacks docstrings, and the method names are also very generic. It's a bit hard to understand what is going on. I think unit tests would be very appropriate and easy to implement.

Reply:
Not really sure unit testing would be easy as it requires a prior run for data, implies an SQLite database and an HTTP server which can be a real pain to test because of latency and would require a new dependency for testing.

I will get on the docstrings right away!

Reply:
Done!

Reply:
If the code is not testable, then there is a problem :|